ee/modules/500-operator-trivy/openapi/config-values.yaml

```diff
@@ -101,10 +109,25 @@ properties:
     -
       - app
       - env
+
   insecureRegistries:
     type: array
-    x-doc-example: [["my.registry.com", "http-only.registry.io"]]
+    x-examples:
+    -
+      - my.registry.com
+      - http-only.registry.io
     items:
       type: string
     description: |
       Container registries to which insecure connections are allowed.
+
+  nonSslRegistries:
```

Comment:
Include this logic into the`insecureRegistries` option

Reply:
done